packages/web/src/services/audius-sdk/audiusSdk.ts

```diff
@@ -34,6 +34,9 @@ const initSdk = async () => {
             }
           )
         },
+        signTransaction: async () => {
+          return 'Not implemented'
```

Comment:
implementing in [this pr](https://github.com/AudiusProject/audius-client/pull/3396/files), just wanted to minimize the changes here